### ISSUE TITLE
Authentication screens lack scrollbar for very low resolution displays

### ISSUE THREAD

**Issue by gerritholl** [vector-im/element-web#19548]
### Steps to reproduce

The registration page is not fully shown on displays with a very low resolution.  There is no scrollbar.  Screenshot:

![no-scroll](https://user-images.githubusercontent.com/500246/139324173-7e59c254-2394-450d-8d22-3c52832861c7.png)

#### What did you expect?

When the registration page cannot be fully shown, there should be a scrollbar to scroll down.

#### What happened instead?

Instead, there is no scroll bar.

### Operating system

Windows and Linux

### Browser information

Chrome 95.  Also tested in Firefox.

### URL for webapp

https://app.element.io/

### Application version

can't find any Settings option, sorry

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by jplatte** [comment]
This affects >15% of desktops according to https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide, I don't think it deserves the "uncommon" label.

**Comment by t3chguy** [comment]
In the cohort of users we see the stat is 12.7% - 
![image](https://user-images.githubusercontent.com/2403652/201908323-4f06edb1-6c0c-4d3b-b5c8-62a349bd9b68.png)

And once we apply our support policy https://github.com/vector-im/element-web#supported-environments to weed out Mobile browsers and the like it drops to 12.3%

The query being
![image](https://user-images.githubusercontent.com/2403652/201908717-ca79278d-3de9-4de4-8279-536cef1484cf.png)

**Comment by jplatte** [comment]
Is that physical screen height (the CSS height required for the register button to be fully visible before was 667px AFAICT)? Anyways I've provided a fix now.

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/201910381-ce8480c0-eb2a-4fe0-9cec-9b95d611284e.png)

Changing the stat to be viewport height < 668 gives the above which is 9.94%

**Comment by jplatte** [comment]
I hope that's enough for `O-Occasional`?

**Comment by t3chguy** [comment]
Given 0 upvotes on the issue it doesn't seem to actually be biting people for whatever reason?

**Comment by jplatte** [comment]
Well I saw it happen yesterday. It was on a Windows computer in a maximized Firefox window. I think a bookmarks toolbar decreased the viewport height, plus default zoom might have been >100% but didn't appear super big and the notebook wasn't super tiny either (probably around 14 inch).
